src/auth/src/lib.rs

```diff
@@ -45,6 +45,9 @@ pub(crate) type BuildResult<T> = std::result::Result<T, build_errors::Error>;
 /// [Credentials]: https://cloud.google.com/docs/authentication#credentials
 pub mod credentials;
 
+/// Auth constants
```

Comment:
optional: remove the comment, as the module name is sufficient

Reply:
good point, done